docs/guides/typescript_project.mdx

```diff
@@ -4,35 +4,26 @@ title: TypeScript Projects
 description: Stricter, safer, and better development experience
 ---
 
-import ApiLink from '@site/src/components/ApiLink';
-
-Crawlee supports TypeScript by covering public APIs with type declarations. This
-allows writing code with auto-completion for TypeScript and JavaScript code alike.
-Besides that, projects written in TypeScript can take advantage of compile-time
-type-checking and avoid many coding mistakes, while providing documentation for
-functions, parameters and return values.
+Crawlee is TypeScript native library, it provides the type definition directly in the package. This allows writing code with auto-completion for TypeScript and JavaScript code alike. Besides that, projects written in TypeScript can take advantage of compile-time type-checking and avoid many coding mistakes, while providing documentation for functions, parameters and return values. It will also help with refactoring a lot.
```

Comment:
```suggestion
Crawlee is built with TypeScript, which means it provides the type definition directly in the package. This allows writing code with auto-completion for TypeScript and JavaScript code alike. Besides that, projects written in TypeScript can take advantage of compile-time type-checking and avoid many coding mistakes, while providing documentation for functions, parameters and return values. It will also help with refactoring a lot, and ensuring the least amount of bugs will sneak through.
```